app/views/downloads/download_linux.html.erb

```diff
@@ -12,49 +12,74 @@
 
   <h3>Debian/Ubuntu</h3>
   <p>For the latest stable version for your release of Debian/Ubuntu</p>
-  <code># apt-get install git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># apt-get install git</pre>
   <p>For Ubuntu, this PPA provides the latest stable upstream Git version</p>
-  <code># add-apt-repository ppa:git-core/ppa</code>
-  <code># apt update; apt install git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># add-apt-repository ppa:git-core/ppa</pre>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># apt update; apt install git</pre>
+  <p>Optionally, if you would also like to install gitk, then run the following:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># sudo apt install gitk</pre>
   
   <h3>Fedora</h3>
-  <code># yum install git</code> (up to Fedora 21)<br>
-  <code># dnf install git</code> (Fedora 22 and later)
+  <p>Fedora 21 and underneath:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># yum install git</pre>
+  <p>Fedora 22 and later:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># dnf install git</pre>
+  <p>Optionally, if you would also like to install gitk, then run the following:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># sudo yum install tk</pre>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># sudo yum install gitk</pre>
 
   <h3>Gentoo</h3>
-  <code># emerge --ask --verbose dev-vcs/git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># emerge --ask --verbose dev-vcs/git</pre>
+  <p>Check this <a href="https://bugs.gentoo.org/171230">Gentoo issue</a> for more information on how to install Gitk on Gentoo</p>
 
   <h3>Arch Linux</h3>
-  <code># pacman -S git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># pacman -S git</pre>
+  <p>Download Git and you will be prompted to download gitk. Check that optional dependency</p>
 
   <h3>openSUSE</h3>
-  <code># zypper install git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># zypper install git</pre>
+  <p>To install gitk:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># zypper install gitk</pre>
 
   <h3>Mageia</h3>
-  <code># urpmi git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># urpmi git</pre>
+  <p>If you would like to optionally install gitk: here is the <a href="https://madb.mageia.org/package/show/application/0/name/gitk">page for gitk</a> on Mageia's package catalog. It has a list of RPM files. Choose your RPM file and then run the following on it:</p>
```

Comment:
I would vouch for keeping this short: if we show where is the package, the user can now find how to install RPM files on their own

Reply:
Sure